xmtp_mls/src/subscriptions/stream_conversations.rs

```diff
@@ -315,7 +334,8 @@ where
                     welcome_envelope?,
                     *this.conversation_type,
                 )?;
-                this.welcome_syncs.spawn(future.process());
+                this.welcome_syncs
+                    .spawn(future.process(*this.include_duplicated_dms));
```

Comment:
you can include the flag on the `ProcessWelcomeFuture` struct, so that you don't need to pass it through all the function layers


then in the filter function it will be available on `self.include_duplicated_dms`, and in the future it will be easier to turn the processfuture into a builder if we add more options similiar to this